src/components/BookList.js

```diff
@@ -0,0 +1,47 @@
+import './Styles/BookList.css';
+import React from 'react';
+import AddNew from './AddNewBook';
+
+const BookList = () => {
+  const books = [
+    {
+      id: 1,
+      booktitl: 'book1',
+      author: 'author1',
+      category: 'category1',
+    },
+    {
+      id: 2,
+      booktitl: 'book2',
+      author: 'author2',
+      category: 'category2',
+    },
+    {
+      id: 3,
+      booktitl: 'book3',
+      author: 'author3',
+      category: 'category3',
+    },
+  ];
+  return (
+    <>
+
+      <div className="book-list">
+        {
+                books.map((book) => (
+                  <div className="book" key={book.id}>
+                    <h3 className="category">{book.category}</h3>
+                    <h2 className="booktitl">{book.booktitl}</h2>
+                    <h3 className="author">{book.author}</h3>
+                    <button type="button" className="btn">Remove</button>
+                  </div>
+                ))
+            }
+
+      </div>
+      <AddNew />
+    </>
+  );
+};
+
```

Comment:
- [ ]  According to the project requirement you should create a component for Contains individual book states and also another component for rendering individual books.
`IndividualBookComponent` -  It may contain details like the book title, author, description, and any other relevant information. Additionally, it should include a "Delete" button that allows the user to delete the book when clicked.
`BookListComponent: ` - This component is responsible for rendering a list of individual books. It will receive an array of book objects as props and iterate through them to display each book.

### Requirement 
![Screenshot 2023-09-22 100604](https://github.com/AlselwiAisha/Bookstore/assets/81727988/6eb67186-89da-4e6f-9479-390b156da80b)